scout/load/panel.py

```diff
@@ -151,6 +174,46 @@ def load_panelapp_panel(adapter, panel_id=None, institute="cust000", confidence=
             raise err
 
 
+def load_panelapp_green_panel(adapter, institute):
+    """Load/Update the panel containing all Panelapp Green genes
+
+    Args:
+        adapter(scout.adapter.MongoAdapter)
+        institute(str): _id of an institute
+
+    """
+    LOG.info("Fetching all panel app panels")
+    panel_ids = _panelapp_panel_ids()
+
+    # check and set panel version
+    old_panel = adapter.gene_panel(panel_id=PANEL_NAME)
+    green_panel = {
+        "panel_name": PANEL_NAME,
+        "display_name": "PanelApp Green Genes",
+        "institute": institute,
+        "version": old_panel["version"] + 0.1 if old_panel else 1,
+        "date": datetime.now(),
+    }
+
+    genes = set()  # avoid duplicate genes from different panels
+    # Loop over all PanelApp panels
+    for _ in panel_ids:
+        # And collect their green genes
+        parsed_panel = _parse_panelapp_panel(adapter, _, institute, "green")
+        genes.update({(gene["hgnc_id"], gene["hgnc_symbol"]) for gene in parsed_panel.get("genes")})
+
+    green_panel["genes"] = [{"hgnc_id": tup[0], "hgnc_symbol": tup[1]} for tup in genes]
+
+    # Do not update panel is new version contains less genes
+    if old_panel and len(old_panel.get("genes", [])) > len(green_panel["genes"]):
```

Comment:
Hm, that could happen from one day to the next, though it is absolutely not likely to be that way with longer update intervals. Add a force flag perhaps?

Reply:
I wasn't convinced of this check myself. Sounds good to keep it like this (think about when something goes wrong and the new panel has no genes) and add a force option, so load it anyway on a second command run!